src/components/layout/ProductsCompare/ProductsCompareContainer.js

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux';
+
+import ProductsCompare from './ProductsCompare';
+
+import { getCompare, addCompare } from '../../../redux/productsRedux';
+
+const mapStateToProps = state => ({
+  compareProducts: getCompare(state),
```

Comment:
`compareProducts` brzmi jak funkcja która porówna produkty, może `productsToCompare`

`getCompare` ta nazwa z kolei jest mało opisowa, nie wiadomo co bierzemy do tego porównania, a może to już pobiera wynik porównania? Pasuje wymyśleć lepszą nazwę ;)

Reply:
git ;)

Reply:
zmienione